pkg/ip/link.go

```diff
@@ -40,9 +41,15 @@ func makeVethPair(name, peer string, mtu int) (netlink.Link, error) {
 	return veth, nil
 }
 
-func makeVeth(name string, mtu int) (peerName string, veth netlink.Link, err error) {
+func makeVeth(namePrefix string, mtu int) (peerName string, veth netlink.Link, err error) {
 	for i := 0; i < 10; i++ {
-		peerName, err = RandomVethName()
+		peerName, err = RandomIfaceName(namePrefix)
```

Comment:
That would mean its no longer a random name though, since FormatChainName() just hashes the input.  We still need some kind of entropy.  If you want to call the same CNI plugin twice on the same container but get different interfaces, then without entropy you'd end up with the same interface name since there's nothing unique about multiple invocations of the plugin.